src/SFA.DAS.ApplyService.Application/Apply/Submit/SubmitApplicationHandler.cs

```diff
@@ -43,9 +41,13 @@ public async Task<bool> Handle(SubmitApplicationRequest request, CancellationTok
                     {
                         application.ApplyData.ApplyDetails.ProviderRoute = request.ProviderRoute;
                     }
-
+                    
                     application.ApplyData.ApplyDetails.ApplicationSubmittedOn = DateTime.UtcNow;
                     application.ApplyData.ApplyDetails.ApplicationSubmittedBy = submittingContact.Id;
+                    foreach(var sequence in application.ApplyData.Sequences)
+                    {
+                        sequence.IsActive = true;
```

Comment:
This looks like it should be set when Creating Application, not submitted

Reply:
Moved in APR-1269 branch